modules/ROOT/nav.adoc

```diff
@@ -3,4 +3,5 @@
 * xref:setup.adoc[Setup Anypoint Visualizer]
 * xref:view.adoc[View Application Information with Anypoint Visualizer]
 * xref:layers.adoc[Use Layers in Anypoint Visualizer]
-* xref:technical.adoc[How Anypoint Visualizer Collects Application Data]
```

Comment:
@valkyrie69 @asimons-mule This file is being removed from the nav, but I don't see the file deleted in the PR.